dags/looker.py

```diff
@@ -153,4 +153,38 @@
         **airflow_gke_prod_kwargs,
     )
 
-    lookml_generator_staging >> lookml_generator_prod
+    looker_folders_to_validate = [
+        706,  # KPI metrics
+    ]
+
+    lookml_validate_spectacles = GKEPodOperator(
+        owner="ascholtz@mozilla.com",
+        email=[
+            "ascholtz@mozilla.com",
+            "telemetry-alerts@mozilla.com",
+        ],
+        task_id="lookml_validate_spectacles",
+        name="lookml-validate-spectacles",
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest",
+        dag=dag,
+        cmds=["bash", "-x", "-c"],
+        arguments=[
+            "pip install spectacles==2.4.10 &&" # todo: remove this once mozilla-nimbus-schemas supports newer pydantic version
+            r"spectacles content --verbose \ "
+            r" --base-url ${BASE_URL} \ "
+            r" --client_id ${LOOKER_API_CLIENT_ID} \ "
+            r" --client_secret ${LOOKER_API_CLIENT_SECRET} \ "
+            r" --project spoke-default \ "
+            r" --branch ${BRANCH} \ "
+            r" --pin-imports looker-hub:main \ "
```

Comment:
Pretty sure these don't need the backslash escapes since this doesn't end up as a mult-line command (and thus the strings also don't need to be raw strings).
```suggestion
            "spectacles content --verbose"
            " --base-url ${BASE_URL}"
            " --client_id ${LOOKER_API_CLIENT_ID}"
            " --client_secret ${LOOKER_API_CLIENT_SECRET}"
            " --project spoke-default"
            " --branch ${BRANCH}"
            " --pin-imports looker-hub:main"
```